.github/workflows/drivers.yml

```diff
@@ -359,3 +359,26 @@ jobs:
       uses: ./.github/actions/test-driver
       with:
         junit-name: 'be-tests-sqlserver-ee'
+
+  be-tests-vertica-ee:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+    env:
+      CI: 'true'
+      DRIVERS: vertica
+    services:
+      vertica:
+        image: sumitchawla/vertica
+        ports:
+          - "5433:5433"
+    steps:
+    - uses: actions/checkout@v3
+    - name: Make plugins directory
+      run: mkdir plugins
+    - name: Fetch JDBC driver
+      run: wget --output-document=plugins/vertica-jdbc-7.1.2-0.jar ${{ secrets.VERTICA_JDBC_JAR }}
```

Comment:
Alternative:
```
curl -OL ${{ secrets.VERTICA_JDBC_JAR }}
mv vertica-jdbc-*.jar plugins/
```

Reply:
Noted.
However, let's deal with this in the next PR. We might end up not even needing this step because EE version comes bundled with Vertica driver.